diesel/src/query_dsl/limit_dsl.rs

```diff
@@ -16,6 +16,36 @@ where
 {
     type Output = <T::Query as LimitDsl>::Output;
 
+    /// Example:
```

Comment:
Can you add a summary line and then use `# Example` as headline? This way it shows up in rustdoc as "Example:" in some places. Maybe something like

> Limit of rows to return

or just

> Set the `LIMIT` on the query